tests/integration/components/pix-options-checkbox-test.js

```diff
@@ -0,0 +1,51 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+import sinon from 'sinon';
+
+module.only('Integration | Component | options-checkbox', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders the default PixOptionsCheckbox', async function(assert) {
+    this.name = "htmlElement";
+    this.value = 'test';
+    // when
+    await render(hbs`
+      <PixOptionsCheckbox 
+        @name={{name}}
+        @value={{value}}>
+        <span id="htmlSpanElement">test</span>
+      </PixOptionsCheckbox>
+    `);
+
+    // then
+    const htmlElement = this.element.querySelectorAll('#htmlSpanElement');
+
+    assert.equal(htmlElement.length, 1);
+    assert.equal(htmlElement.item(0).textContent, 'test');
+  });
+
+  test('it should trigger onChange function when an item is selected', async function (assert) {
+    // given
+    this.name = "htmlElement";
+    this.value = 'span';
+    this.onChange = sinon.spy();
+  
+    await render(hbs`
+      <PixOptionsCheckbox 
+        @name={{name}}
+        @value={{value}}
+        @onChange={{onChange}}>
+        <span id="htmlSpanElement"/>
```

Comment:
Un span est pas auto-fermant il me semble :
```suggestion
        <span id="htmlSpanElement">test</span>
```